views/layout.pug

```diff
@@ -3,5 +3,24 @@ html
   head
     title= title
     link(rel='stylesheet', href='/stylesheets/style.css')
+    link(rel='stylesheet', href='https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css')
```

Comment:
On one hand, bootstrap makes things easy, on the other, it looks sorta generic...
Did the design team approve of this?